qmcpy/algorithms/function/__init__.py

```diff
@@ -1,69 +0,0 @@
-''' Originally developed in MATLAB by Fred Hickernell. Translated to python by Sou-Cheng T. Choi and Aleksei Sorokin '''
```

Comment:
I have definitely seen projects that use the `__init__.py` to do stuff, but usually it's just to surface content from the rest of the directory.  I would personally frown on the idea of important classes that will be used elsewhere being uniquely defined here.

Reply:
Should we move the abstract classes to  files of the same name?
For example, should we put the Integrand abstarct class at qmcpy/algorithms/integrand/Integrand.py? maybe .../IntegrandBase.py?